cellranger/3.1/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:18.04
+MAINTAINER Alex Sickler (sicklera@email.chop.edu)
+
+ENV CELLRANGER_VERSION 3.1
+
+#before building, get the link to download from:
+#https://support.10xgenomics.com/single-cell-gene-expression/software/downloads/latest
+ARG cellranger_uri
+
+RUN apt update && apt install -y wget; \
+wget -O cellranger-3.1.0.tar.gz $cellranger_uri; \
```

Comment:
Not necessary but if you set CELLRANGER_VERSION to 3.1.0 you can substitute it in here. `cellranger-${CELLRANGER_VERSION}.tar.gz`. It will make it easier to update in the future.